### PR TITLE
Doc adjust

### PR DESCRIPTION
**Pull request by joedragons**
This pull request makes the following changes:
* Link adjustments
** Some text adjustments to reflect what the (new) links show
* Typo fix

### CODE REVIEW

File: docs/R_USER_DOCUMENTATION.md [hunk 1]
```diff
@@ -37,22 +37,20 @@ a hosted repository.
 You can set up an R proxy repository to access a remote repository location, for example the official R
 registry at [https://cran.r-project.org/](https://cran.r-project.org/) that is configured as the default in R.
 
-To proxy a R repository, you simply create a new 'r (proxy)' as documented in [Repository Management](https://help.sonatype.com/display/NXRM3/Configuration#Configuration-RepositoryManagement) in
-details. Minimal configuration steps are:
+To proxy a R repository, you simply create a new 'r (proxy)' as shown in the documented example in
```
Comment (ataylor284) [inline docs/R_USER_DOCUMENTATION.md]: Pedantic: I hate when documentation implies something should be simple when it is often not.  I'd rephrase it without "simply".

Comment (joedragons) [reply docs/R_USER_DOCUMENTATION.md]: Will boyscout, no-one can stop me!=D

**Comment by joedragons** [comment]
Not sure why I had to merge master as https://github.com/sonatype-nexus-community/nexus-repository-r/pull/33/commits/6b812580c866a4efd5ee270af99923cf5e8dc875 shows clean but whatever.
